scavenger-agent-java/src/integrationTest/java/integrationTest/support/AgentRunner.java

```diff
@@ -0,0 +1,85 @@
+package integrationTest.support;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+
+public class AgentRunner implements Callable<String> {
+    private final String javaPath;
+    private final String classpath;
+    private final String agentPath;
+
+    private String configFilePath;
+    private String cmdLineArgs;
+
+    public AgentRunner(String javaPath, String classpath, String agentPath) {
+        this.javaPath = javaPath;
+        this.classpath = classpath;
+        this.agentPath = agentPath;
+    }
+
+    public void setConfigFilePath(String configFilePath) {
+        this.configFilePath = configFilePath;
+    }
+
+    public void setConfig(Properties properties) {
+        cmdLineArgs = properties.entrySet().stream()
+            .map(entry -> entry.getKey() + "=" + entry.getValue())
+            .collect(Collectors.joining(";"));
+    }
+
+    @Override
+    public String call() throws IOException, InterruptedException, RuntimeException {
+        List<String> command = buildCommand();
+        Process process = new ProcessBuilder().command(command).redirectErrorStream(true).start();
+        String output = collectProcessOutput(process.getInputStream());
+        int exitCode = process.waitFor();
+        if (exitCode != 0) {
+            throw new RuntimeException(String.format("Could not execute '%s': %s%nExit code=%d", command, output, exitCode));
```

Comment:
Do not use format...(It require parsing symbols...)
just use + for string construction

Reply:
fixed in [fe07c0f](https://github.com/naver/scavenger/pull/71/commits/fe07c0fa0198503b576deb9b6af3a69701702650).